src/components/Book.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+function Book() {
+  return (
+    <>
+      <div className="books">
+        <h2 className="book-title">The Hunger Games</h2>
+        <p className="book-author">Suzanne Collins</p>
+        <button type="button" className="remove-btn">Remove</button>
+      </div>
+    </>
+  );
```

Comment:
- [ ] Please ensure that the `Book` component takes the book data as a prop and renders the title and the author. This component would be used in the `Books` component 😎

Reply:
Thank you for the highlights.